src/include/miopen/conv/problem_description.hpp

```diff
@@ -387,37 +390,41 @@ struct ProblemDescription : ProblemDescriptionBase
 #if MIOPEN_ENABLE_SQLITE
     static std::string table_name() { return "config"; }
 
-    template <class Self, class F>
-    static void Visit(Self&& self, F f)
+    template <class Self>
+    static void Visit(Self&& self, std::function<void(int, std::string)> f)
```

Comment:
[Q] I am afraid that changing this function breaks compatibility with existing databases.

Or was this function never used but _will_ be used after this PR instead of `ProblemDescription::Visit()`?

Reply:
Resolved.